src/families/bitcoin/wallet-btc/wallet.ts

```diff
@@ -272,9 +272,7 @@ class BitcoinLikeWallet {
       if (additionals && additionals.includes("peercoin")) {
         // remove timestamp for new version of peercoin input, refer to https://github.com/peercoin/rfcs/issues/5 and https://github.com/LedgerHQ/ledgerjs/issues/701
         const version = i.txHex.substring(0, 8);
-        if (version !== "01000000" && version !== "02000000") {
-          hasTimestamp = false;
-        }
+        hasTimestamp = version === "01000000" || version === "02000000";
```

Comment:
This always overrides initial value of `params.hasTimestamp` which becomes never used.
Just to mention, but I think this is ok since Peercoin is the only remaining supported coin that's using timestamp...

Reply:
but only in case of peercoin